### PR TITLE
Replace obsolete Tile Access methods

### PR DESCRIPTION
**Pull request by MilenVolf**
## About the PR

I tried to make it more clean and sticked every method replacement to separate commits to make forks life a little easier.

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

### CODE REVIEW

File: Content.Server/Procedural/DungeonSystem.Commands.cs [hunk 1]
```diff
@@ -44,7 +42,8 @@ private async void GenerateDungeon(IConsoleShell shell, string argstr, string[]
         }
 
         var position = new Vector2i(posX, posY);
-        var dungeonUid = _mapManager.GetMapEntityId(mapId);
+        _maps.TryGetMap(mapId, out var dungeonUidNullable);
+        var dungeonUid = dungeonUidNullable!.Value;
```
Comment (PJB3005) [inline Content.Server/Procedural/DungeonSystem.Commands.cs:46]: You're using a Try* version of the fetch, but then ignoring the success status and assuming it succeeded. What you should instead be doing is one of these:

* Use `GetMap()` directly if you're certain the map exists.
* Use `GetMapOrInvalid()` to preserve the existing behavior.

Do the latter, as I don't want to have warning fix PRs like this possibly changing behavior.

File: Content.Server/Procedural/DungeonSystem.Commands.cs [hunk 2]
```diff
@@ -118,7 +117,8 @@ private void DungeonPackVis(IConsoleShell shell, string argstr, string[] args)
         }
 
         var mapId = new MapId(mapInt);
-        var mapUid = _mapManager.GetMapEntityId(mapId);
+        _maps.TryGetMap(mapId, out var mapUidNullable);
+        var mapUid = mapUidNullable!.Value;
```
Comment (PJB3005) [inline Content.Server/Procedural/DungeonSystem.Commands.cs:121]: Ditto.

File: Content.Server/Procedural/DungeonSystem.Commands.cs [hunk 3]
```diff
@@ -174,7 +174,8 @@ private void DungeonPresetVis(IConsoleShell shell, string argstr, string[] args)
...
```
Comment (PJB3005) [inline Content.Server/Procedural/DungeonSystem.Commands.cs:178]: Ditto.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: Content.Server/Procedural/DungeonSystem.Commands.cs (hunk 1 above)
Comment (MilenVolf) [reply Content.Server/Procedural/DungeonSystem.Commands.cs:46]: Yeah, I thought about `GetMapOrInvalid` too, but wasn't sure if I should use it. Will fix it

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**